ui/src/app/google-chart/google-chart.service.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@angular/core';
+
+declare var google: any;
+
+/**
+ * Service to access Google Chart library
+ */
+@Injectable({
+  providedIn: 'root'
+})
+export class GoogleChartService {
+
+  private readonly google: any;
+  private loaderPromise: Promise<any>;
+  number = 1;
+
+  constructor() {
+    this.google = google;
+  }
+
+  getGoogle(): any {
```

Comment:
Is this still needed?

Reply:
I think it is still needed for library access in ``DonutChartComponent`` ?